libs/components/datetime/src/lib/modules/datepicker/datepicker-calendar-inner.component.ts

```diff
@@ -129,16 +134,43 @@ export class SkyDatepickerCalendarInnerComponent
   ];
 
   #ngUnsubscribe = new Subject<void>();
+  #prevDay: string | undefined;
+  #nextDay: string | undefined;
+  #prevMonth: string | undefined;
+  #nextMonth: string | undefined;
+  #prevYear: string | undefined;
+  #nextYear: string | undefined;
 
   #_selectedDate: Date | undefined;
   #_startingDay = 0;
 
+  readonly #resourcesSvc = inject(SkyLibResourcesService);
+
   public ngOnInit(): void {
     if (this.selectedDate) {
       this.activeDate = new Date(this.selectedDate);
     } else {
       this.activeDate = new Date();
     }
+
+    this.#resourcesSvc
+      .getStrings({
+        prevDay: 'skyux_datepicker_move_calendar_previous_day',
+        nextDay: 'skyux_datepicker_move_calendar_next_day',
+        prevMonth: 'skyux_datepicker_move_calendar_previous_month',
+        nextMonth: 'skyux_datepicker_move_calendar_next_month',
+        prevYear: 'skyux_datepicker_move_calendar_previous_year',
+        nextYear: 'skyux_datepicker_move_calendar_next_year',
+      })
+      .pipe(takeUntil(this.#ngUnsubscribe))
+      .subscribe((resources) => {
+        this.#prevDay = resources.prevDay;
+        this.#nextDay = resources.nextDay;
+        this.#prevMonth = resources.prevMonth;
+        this.#nextMonth = resources.nextMonth;
+        this.#prevYear = resources.prevYear;
+        this.#nextYear = resources.nextYear;
```

Comment:
Won't the view need to be refreshed when new strings are returned from the resources service? There are a few async behaviors in this component that could affect the view; maybe rethink how they should all interact when one or more of them change?

Reply:
I think just adding a call to `this.refreshView()` should handle everything. On calendar instantiation, it won't run if the child view hasn't been initalized, but then everything other change detection cycle runs through it so these resource strings should too.